k9mail/src/main/java/com/fsck/k9/preferences/DeletePolicy.java

```diff
@@ -0,0 +1,34 @@
+package com.fsck.k9.preferences;
+
+import android.util.Log;
+
+import com.fsck.k9.Account;
+import com.fsck.k9.K9;
+import com.fsck.k9.mail.ServerSettings;
+import com.fsck.k9.mail.store.imap.ImapStore;
+import com.fsck.k9.mail.store.pop3.Pop3Store;
+import com.fsck.k9.mail.store.webdav.WebDavStore;
+
+/**
+ * Defaults for mail deletion.
+ */
+public class DeletePolicy {
```

Comment:
welcome to the K9-refactor-:rabbit:-hole :)

hmm, `AccountManager` is generic and also overloaded.. `AccountHandler` /  `AccountCreator` /  `AccountConfigurator` / `AccountMaker` ?